api/clients/v2/verification/blob_verifier.go

```diff
@@ -0,0 +1,74 @@
+package verification
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda/common"
+
+	disperser "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
+	verifierBindings "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDABlobVerifier"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+// BlobVerifier is responsible for making eth calls against the BlobVerifier contract to ensure cryptographic and
```

Comment:
add link to BlobVerifier contract?

Reply:
Do you mean like this? I don't like that this sort of link is instantly out of date, but I don't know of a better way to link here

https://github.com/Layr-Labs/eigenda/blob/06e88b311ef27618b4b878a78cbbb24358104ab9/contracts/src/core/EigenDABlobVerifier.sol#L108

Reply:
Yes did mean like this. We can also just link to the contract itself, in the master branch, so that it doesn't get out of date instantly.

Reply:
Added [8bba429a](https://github.com/Layr-Labs/eigenda/pull/1066/commits/8bba429aea90afe9366b87d94cb961c6eac459b4)